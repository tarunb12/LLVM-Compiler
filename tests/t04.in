// Variable assignment test

int main() {
    int x = 5
    bool b = x == 4
    if (b == true) {
        string string_of_bool = "true"
        printf("x == %d: %s", 4, string_of_bool)
    }
    else {
        string string_of_bool = "false"
        printf("x == %d: %s", 4, string_of_bool)
    }
    return 0
}