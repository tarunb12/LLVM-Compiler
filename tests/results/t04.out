x == 4: false